### PR TITLE
Add math.comb function

### PR DESCRIPTION
**Pull request by voidsatisfaction**
I implemented `math.comb` function.

As I am new to Rust, if there is any other suggestions, please give me some advice :)

### CODE REVIEW

File: vm/src/stdlib/math.rs [hunk 1]
```diff
@@ -510,6 +510,43 @@ fn math_factorial(value: PyIntRef, vm: &VirtualMachine) -> PyResult<BigInt> {
...
+    let n = n.as_bigint();
+    let one = BigInt::one();
+    let zero = BigInt::zero();
+
+    if n.is_negative() || k.is_negative() {
+        return Err(vm.new_value_error("comb() not defined for negative values".to_owned()))
+    }
+
+    let temp = n - k;
+    if temp < zero {
```
Comment (youknowone) [inline vm/src/stdlib/math.rs:524]:
```suggestion
    if temp.is_negative() {
```

File: vm/src/stdlib/math.rs [hunk 2]
```diff
@@ -510,6 +510,43 @@ fn math_factorial(value: PyIntRef, vm: &VirtualMachine) -> PyResult<BigInt> {
...
+        return Ok(zero);
+    }
+
+    if temp < *k {
+        k = &temp
+    }
+
+    if *k == zero {
```
Comment (youknowone) [inline vm/src/stdlib/math.rs:532]:
```suggestion
    if k.is_zero() {
```

**Review by youknowone — CHANGES_REQUESTED** [review]

File: vm/src/stdlib/math.rs (hunk 1 above)
Comment (voidsatisfaction) [reply vm/src/stdlib/math.rs]: I fixed it. Thanks!

File: vm/src/stdlib/math.rs (hunk 2 above)
Comment (voidsatisfaction) [reply vm/src/stdlib/math.rs]: I fixed it. Thanks!

**Review by youknowone — APPROVED** [review]
looks good, thank you for contributing!

**Pull request merged**